sunpy/timeseries/timeseries_factory.py

```diff
@@ -377,7 +377,8 @@ def _parse_args(self, *args, **kwargs):
         # same order as the input, currently they are not.
         return data_header_unit_tuples, data_header_pairs, already_timeseries, filepaths
 
-    def __call__(self, *args, **kwargs):
+    def __call__(self, *args, concatenate=False, silence_errors=False,
```

Comment:
Python 2.7 is sad because you can only do this (mixing args, kwargs, and specific keyword arguments) in python 3 I think.

Reply:
Well do you want this in for Sunpy 0.9.x or are you happy to wait for 1.0?

Reply:
Is there still time to get stuff into 0.9?

Reply:
We can probably do a back port to 0.9.1.

Reply:
I would rather not, given it's pretty API twiddily

Reply:
Yeh, that's what I thought, it is technically is an API change because users can't just pass whatever kwargs they want to the functions I've removed **kwargs from (which is a good thing!).